testsuite/features/reposync/srv_sync_channels.feature

```diff
@@ -82,6 +82,16 @@ Feature: Be able to list available channels and enable them
     And I should get "[I] RES-AS-8-Updates for x86_64 SUSE Linux Enterprise Server with Expanded Support 8 x86_64 [res-as-8-updates-x86_64]"
     And I should get "[I] RES-CB-8-Updates for x86_64 SUSE Linux Enterprise Server with Expanded Support 8 x86_64 [res-cb-8-updates-x86_64]"
 
+@susemanager
+  Scenario: Enable SLES15 client tools for creating bootstrap repositories
```

Comment:
It seems we also specify SP4 in the channel, so better have a title like "SLES15 SP4 client tools"